From 7fe4c488149b7fe6ab1353675fc2e0c1ddc5099d Mon Sep 17 00:00:00 2001
From: Jingyun Hua <huajingyun@loongson.cn>
Date: Tue, 26 Nov 2024 08:25:08 +0000
Subject: [PATCH] setup-desktop: remove consolekit2

consolekit2 was removed last month, ref:
https://git.alpinelinux.org/aports/commit/?id=a2f2d8f15fba
https://git.alpinelinux.org/aports/commit/?id=340ad0e2c5a7

Maybe it's time to remove consolekit2 from setup-desktop.in, as mentioned in
https://git.alpinelinux.org/aports/commit/?id=340ad0e2c5a7, mate
doesn't seem to need it, mate can work without consolekit2.
---
 setup-desktop.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup-desktop.in b/setup-desktop.in
index a2aa668..7a94fc9 100644
--- a/setup-desktop.in
+++ b/setup-desktop.in
@@ -56,7 +56,6 @@ case "$de" in
 		;;
 	mate)
 		setup-xorg-base mate-desktop-environment ${BROWSER:-firefox} \
-			consolekit2 \
 			gvfs \
 			lightdm \
 			lightdm-gtk-greeter \
@@ -70,7 +69,6 @@ case "$de" in
 		;;
 	xfce)
 		setup-xorg-base xfce4 ${BROWSER:-firefox} \
-			consolekit2 \
 			gvfs \
 			lightdm \
 			lightdm-gtk-greeter \
-- 
2.43.0

